INSTALL.md

```diff
@@ -11,24 +11,45 @@
 
     Examples: `/home/myuser/www/web/` or `/var/sites/<project-name>/web/`
 
-## Clean install
+## Prerequisite
 
-1. You can get eZ Platform using composer with the following commands:
+  These instructions assume you have strong technical knowledge and have already installed PHP, web server & a database server with a corresponding clean database needed for this software.
+  For further information on requirements see: https://doc.ez.no/display/EZP/Requirements
+
+## Install
+
+1. **Get eZ Platform**:
+
+    A. **Archive** (tar/zip)
+
+       Extract the eZ Platform 2015.01(or higher) archive to a directory, then execute post install scripts:
+
+       *Note: The post install scripts will ask you to fill in some settings, including database settings.*
+
+       ```bash
+       cd /<directory>/
+       curl -s http://getcomposer.org/installer | php
+       php -d memory_limit=-1 composer.phar run-script post-install-cmd
+       ```
+
+
+    B. **Composer**
+
+     You can get eZ Platform using composer with the following commands:
+
+     *Note: composer will take its time to download all libraries and when done you will be asked to fill in some settings, including database settings.*
 
        ```bash
        curl -s http://getcomposer.org/installer | php
        php -d memory_limit=-1 composer.phar create-project --no-dev --prefer-dist ezsystems/ezpublish-community <directory> <version>
        cd /<directory>/
        ```
-       
-      Options:
-      - `<version>`: `dev-master` to get current development version (pre release), `v2015.01` to pick a specific release, otherwise skip it to get latest stable release.
```

Comment:
I suggest you adapt this part when merging to ezplatform repo to reflect the tag name you'll use